### PR TITLE
CED-745: moved some stuff around, added mem for pricing

### PR DESCRIPTION
**Pull request by nravic**
## Describe your changes
Added check for CEDANA_URL, made sure endpoint for gcloud service account was valid, added some more attributes to pubsub data. 
## Checklist before requesting a review
- [ ] Have I performed a self-review?
- [ ] Have I added regression or unit tests (where it makes sense) for my changes?
- [ ] Have I updated the README if changes have resulted in it being out of date?
- [ ] Have I checked to ensure my PR only introduces the changes it's intended to? E.g. No extraneous formatting changes.
- [ ] Have I ensured that there are no performance regressions by checking benchmark results?
- [ ] Is this a breaking change? If yes, please describe areas affected and request reviews from developer stakeholders.

### CODE REVIEW

File: cmd/daemon.go [hunk 1]
```diff
@@ -110,7 +112,7 @@ func gcloudAdcSetup(ctx context.Context) error {
 		// already present skip
 		return nil
 	}
-	url := getenv("CEDANA_SERVER_URL", "https://sandbox.cedana.ai") + "/k8s/gcloud/serviceaccount"
+	url := getenv("CEDANA_URL", "https://sandbox.cedana.ai") + "/k8s/gcloud/serviceaccount"
```
Comment (yashanand1910) [inline cmd/daemon.go:115]: Should use viper to fetch this value instead of directly from env. It might exist in the config.

File: cmd/daemon.go [hunk 2]
```diff
@@ -165,6 +173,11 @@ func pollForAsrMetricsReporting(ctx context.Context, port uint32) {
 			// Publish a message
 			result := topic.Publish(ctx, &pubsub.Message{
 				Data: b,
+				Attributes: map[string]string{
```
Comment (yashanand1910) [inline cmd/daemon.go:176]: Is this data not fetchable from Cadvisor itself?

**Review by yashanand1910 — CHANGES_REQUESTED** [review]
Small fixes needed. Otherwise lgtm.

**Review by bsmithai — APPROVED** [review]
lgtm

**Pull request merged**